benchmark/wildtime_benchmarks/data_generation_yearbook.py

```diff
@@ -37,21 +37,33 @@ def __init__(self, data_dir: str):
         self._dataset = datasets
         self.data_dir = data_dir
 
```

Comment:
The same comment about the pre-defined split applies here - we should validate this.

Reply:
It's also a 10:1 ratio, well balanced!